eva-repository/src/main/kotlin/com/razz/eva/repository/JooqEventRepository.kt

```diff
@@ -171,4 +171,23 @@ class JooqEventRepository(
             }
         }
     }
+
+    private fun String.utf8SizeInBytes(): Int {
```

Comment:
leetcode yellow, I'm glad I brought you here

Reply:
:prytkov: :small_toxic: